packages/thirdparty-widgets/src/Lottie/index.tsx

```diff
@@ -0,0 +1,417 @@
+import { AlphaLottieHtml, AlphaLottieReact } from "@brizy/builder-ui";
+import { Renderer } from "@brizy/builder-ui/lib/elements/AlphaLottie/utils";
+import { Brizy } from "@brizy/core";
+import { HOVER, NORMAL } from "../utils/stateMode";
+
+interface Props {
+  renderer: "svg" | "canvas";
+  autoplay: "off" | "on";
+  direction: -1 | 1;
+  loop: "on" | "off";
+  speed: number;
+}
+
+export const LottieEditor = ({ renderer, autoplay, loop, speed, direction }: Props) => {
+  const isAutoplay = autoplay === "on";
+  const isLoop = loop === "on";
+
+  return (
+    <AlphaLottieReact
+      animationData="https://assets6.lottiefiles.com/private_files/lf30_1KyL2Q.json"
+      renderer={renderer === "svg" ? Renderer.svg : Renderer.canvas}
+      isAutoplay={isAutoplay}
+      isLoop={isLoop}
+      direction={direction}
+      speed={speed}
+    />
+  );
+};
+
+export const LottiePreview = ({ renderer, autoplay, loop, speed, direction }: Props) => {
+  const isAutoplay = autoplay === "on";
+  const isLoop = loop === "on";
+
+  return (
+    <AlphaLottieHtml
+      animationData="https://assets6.lottiefiles.com/private_files/lf30_1KyL2Q.json"
+      renderer={renderer === "svg" ? Renderer.svg : Renderer.canvas}
+      isAutoplay={isAutoplay}
+      isLoop={isLoop}
+      direction={direction}
+      speed={speed}
+    />
```

Comment:
`AlphaLottieReact` and `AlphaLottieHtml` receive the same props, combine in 1 component like you did it in `Map`